examples/server.cc

```diff
@@ -1168,7 +1166,16 @@ void Handler::on_stream_close(uint32_t stream_id) {
 }
 
 namespace {
-void swritecb(struct ev_loop *loop, ev_io *w, int revents) {}
+void swritecb(struct ev_loop *loop, ev_io *w, int revents) {
+  auto s = static_cast<Server *>(w->data);
+
+  auto rv = s->on_write();
+  if (rv == NETWORK_ERR_SEND_NON_FATAL) {
+    ev_io_start(loop, w);
```

Comment:
It looks like there is no ev_io_stop other than disconnect().  This might cause busy loop.